zerver/lib/integrations.py

```diff
@@ -264,6 +264,13 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
     WebhookIntegration('beanstalk', ['version-control'], stream_name='commits'),
     WebhookIntegration('basecamp', ['project-management']),
     WebhookIntegration('beeminder', ['misc'], display_name='Beeminder'),
+    WebhookIntegration(
+        'bitbucket3',
+        ['version-control'],
+        logo='static/images/integrations/logos/bitbucket.svg',
+        display_name='Bitbucket Server',
```

Comment:
It is `BitBucket`, not `Bitbucket`

Reply:
Actually, I think that it *is* "Bitbucket" and not "BitBucket" :slightly_smiling_face: . I remember that I made the mistake of typing out "BitBucket" a couple of times in zulipchat but then I started correcting this mistake after finding out.

sources:
1. https://bitbucket.org/product/
2. https://en.wikipedia.org/wiki/Bitbucket